### PR TITLE
ansible-lint (documentation) cleanup for plugins/

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

Fixes an array of ansible-lint failures in plugins/
Adds `ansible-lint plugins/` to `tox -m lint`

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/

##### ADDITIONAL INFORMATION

docs changes only (no changelog fragment needed)

### CODE REVIEW

**Comment by tremble** [comment]
see also:
https://github.com/ansible/ansible-lint/issues/3860

File: plugins/lookup/secretsmanager_secret.py [hunk 1]
```diff
@@ -78,39 +78,39 @@
...
-     instance_name: app-db
-     db_engine: MySQL
-     size: 10
-     instance_type: db.m1.small
-     username: dbadmin
-     password: "{{ lookup('amazon.aws.aws_secret', 'DbSecret') }}"
-     tags:
-       Environment: staging
+- name: Create RDS instance with aws_secret lookup for password param
+  rds:
```
Comment (alinabuzachis) [inline plugins/lookup/secretsmanager_secret.py:85]: Is this example still valid?

**Review by alinabuzachis — APPROVED** [review]

**Comment by tremble** [comment]
regate

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Pull request merged**